src/commands/moderation/warn/warn.ts

```diff
@@ -0,0 +1,149 @@
+import { TextChannel, User } from 'discord.js';
+import moment from 'moment';
+import { CommandDefinition } from '../../../lib/command';
+import { Roles, Channels, CommandCategory } from '../../../constants';
+import { makeEmbed } from '../../../lib/embed';
+import { getConn } from '../../../lib/db';
+
+const permittedRoles = [
+    Roles.ADMIN_TEAM,
+    Roles.MODERATION_TEAM,
+];
+
+const warnEmbed = (user: User) => makeEmbed({
+    title: `${user.tag} was warned successfully`,
+    color: 'GREEN',
+});
+
+const modLogEmbed = (formattedDate, moderator: User, user: User, reason: string) => makeEmbed({
+    author: {
+        name: `[warned]  ${user.tag}`,
```

Comment:
```suggestion
        name: `[WARNED]  ${user.tag}`,
```
Better appearance on discord.